R/tm_t_pp_prior_medication.R

```diff
@@ -72,16 +72,27 @@ template_prior_medication <- function(dataname = "ANL",
 #' @export
 #'
 #' @examples
-#' adcm <- tmc_ex_adcm
-#' adsl <- tmc_ex_adsl %>% dplyr::filter(USUBJID %in% adcm$USUBJID)
-#' adcm$CMASTDTM <- adcm$ASTDTM
-#' adcm$CMAENDTM <- adcm$AENDTM
+#' ADCM <- tmc_ex_adcm
+#' ADSL <- tmc_ex_adsl %>% dplyr::filter(USUBJID %in% ADCM$USUBJID)
+#' ADCM$CMASTDTM <- ADCM$ASTDTM
+#' ADCM$CMAENDTM <- ADCM$AENDTM
 #' adcm_keys <- c("STUDYID", "USUBJID", "ASTDTM", "CMSEQ", "ATC1", "ATC2", "ATC3", "ATC4")
 #'
 #' app <- init(
 #'   data = cdisc_data(
-#'     cdisc_dataset("ADSL", adsl),
-#'     cdisc_dataset("ADCM", adcm, keys = adcm_keys)
+#'     ADSL = ADSL,
+#'     ADCM = ADCM,
+#'     code = "
+#'       ADCM <- tmc_ex_adcm
+#'       ADSL <- tmc_ex_adsl %>% dplyr::filter(USUBJID %in% ADCM$USUBJID)
+#'       ADCM$CMASTDTM <- ADCM$ASTDTM
+#'       ADCM$CMAENDTM <- ADCM$AENDTM
+#'     ",
+#'     join_keys = join_keys(
+#'       join_key("ADSL", keys = get_cdisc_keys("ADSL")),
+#'       join_key("ADCM", keys = adcm_keys),
+#'       join_key("ADCM", "ADSL", keys = c("USUBJID", "STUDYID"))
+#'     )
```

Comment:
Please so the same as mentioned in previous comment. Please make sure `get_cdisc_keys` isn't used in the package
```suggestion
#'     join_keys = join_keys,
```